src/types/bridge.ts

```diff
@@ -0,0 +1,46 @@
+export interface BridgeTx extends BridgeTxResult {
+}
+
+export interface BridgeTxResult {
+    hash:     string;
+    height:   string;
+    index:    number;
+    tx_result: BridgeResponseTx;
+    tx:       string;
+}
+
+export interface BridgeResponseTx {
+    code:                 number;
+    data?:                 null;         
+    log?:                  string;        
+    info?:                 string;          
+    gasWanted:            string;          
+    gasUsed:              string;          
+    events:               BridgeMsg[];
+    codespace:            string;
+    XXX_NoUnkeyedLiteral?: string;
```

Comment:
Ditto on the `XXX_`-prefixed keys here

Reply:
https://github.com/denalimarsh/amino-js/pull/2/commits/cecdc355c128f39ca475006791031e59fdb102a3